examples/ethfs/.gitignore

```diff
@@ -3,4 +3,5 @@ node_modules/
 
 .env.local
 .ponder/
-generated/
```

Comment:
Would be good to convert all the examples to use the .gitignore that I pasted into the create-ponder script

Reply:
Your example .gitignore doesn't include .ponder